<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="Daniel A. Cuevas">

    <title>Daniel Cuevas</title>

    <!-- Bootstrap Core CSS -->
    <link href="assets/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="assets/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="assets/css/bootstrap-social.css" rel="stylesheet" type="text/css">
    <link href="assets/css/academicons.css" rel="stylesheet" type="text/css">

    <!-- Custom CSS -->
    <link href="assets/css/full-width-pics.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker|Montserrat|Roboto" rel="stylesheet">

    <!-- Tab Icon -->
    <link rel="shortcut icon" type="image/x-icon" href="assets/imgs/initials.png" />

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

    <style>
        #main {
            padding: 10px 25px;
            text-align: left;
            line-height: 25px;
            font-size: 16px;}
        h3 {
            font-weight: bolder;
            color: #004173;}
        ul.ul-text, ol {
            margin-right: 175px;}
        ol {
            margin-left: -1.5em;}
        .text1 {
            margin-left: 25px;}
        .dates {
            float: right;
            font-style: italic;}
        .jobpos {
            font-style: italic;}
    </style>

</head>

<body>

    <!-- Navigation -->
    <nav class="navbar navbar-inverse navbar-custom navbar-fixed-top" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="navbar-brand" href="index.html">Daniel Cuevas</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="about.html">About</a>
                    </li>
                    <li>
                        <a href="research.html">Research</a>
                    </li>
                    <li>
                        <a href="cv.html">Curriculum Vitae</a>
                    </li>
                    <li class="contact-button">
                        <a href="contact.html">Contact</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    <!-- Full Width Image Header with Logo -->
    <!-- Image backgrounds are set within the full-width-pics.css file. -->
    <header class="image-bg-fluid-height">
        <div class="container">
            <div class="row">
                <div id="main" class="col-xs-12">
                    <div>
                        <a href="cv.pdf">Download PDF</a>
                    </div>
                    <h3>EDUCATION</h3>
                    <div class="text1">
                        <b>Doctor of Philosophy (PhD), Computational Biology and Bioinformatics</b><span class="dates">08/12-02/18</span><br/>
                        San Diego State University with Claremont Graduate University<br/>
                        <br/>
                        <b>Bachelor of Science (BS), Computer Science</b><span class="dates">08/05-05/11</span><br/>
                        San Diego State University<br/>
                    </div>

                    <h3>GRADUATE RESEARCH</h3>
                    <div class="text1">
                        Edwards Bioinformatics Lab&nbsp;<a href="https://edwards.sdsu.edu/research/" target="_blank"><i class="fa fa-external-link-square"></a></i><br/>
                        San Diego State University<br/>
                        5500 Campanile Drive, San Diego, CA 92182<br/>
                        <b>Mentor:</b> Professor Robert A. Edwards
                    </div>

                    <h3>PROFESSIONAL EXPERIENCE</h3>
                    <div class="text1">
                        <b>Bioinformatics Scientist II</b><span class="dates">03/18-Present</span><br/>
                        <span class="jobpos">Progenity, R&amp;D<br/></span>
                        <ul class="ul-text">
                            <li>Led development and design of single-molecule detection assay analytical software, processing daily experiments as an integral piece for rapid results during early stage product development</li>
                            <li>Developed probabilistic classifier to distinguish between levels of NASH liver disease severity and fibrosis measures in bisulfite sequence samples using key differentially methylated DNA loci, achieving project sensitivity and precision targets</li>
                            <li>Led development and design of paired-end shallow sequencing NIPT analytical software, tests for autosomal and sex chromosome aneuploidies, microdeletions, and fetal fraction prediction, achieving MVP and TPP targets</li>
                            <li>Implemented key pre-processing and QC components for the Preeclampsia diagnostic rule-out test analytical software, achieving software performance validation at each product milestone</li>
                            <li>Established the in-house microbiome 16S analysis pipeline as lead bioinformatics scientist for microbiome analyses</li>
                            <li>Created and managed R&D pipelines across several projects (NASH, Preeclampsia, NIPT, single-molecule detection platform), ensuring daily experiments underwent quality control, pre-processing, and primary analysis within 1-day turnaround times</li>
                        </ul><br/>

                        <b>Graduate Research Assistant</b><span class="dates">07/12-03/18</span><br/>
                        <span class="jobpos">Robert Edwards Bioinformatics Lab, San Diego State University<br/></span>
                        <ul class="ul-text">
                            <li>Led production of PyFBA, a genome-scale metabolic model builder using  phenotypic-based assays, to identify novel metabolic pathways and transporters within bacterial genomes</li>
                            <li>Developed PMAnalyzer to profile bacterial phenotypes across thousands of growth curves from monoculture and microbial community assays using a logistic growth model fit to absorbance data</li>
                            <li>Participated in design and development of a full NGS metagenome and metatranscriptome computational pipeline used for emergency Cystic Fibrosis patient sputum sequencing data</li>
                        </ul><br/>

                        <b>Bioinformatics Scientist and Software Engineer</b><span class="dates">09/11-07/12</span><br/>
                        <span class="jobpos">Life Technologies, Ion Torrent R&amp;D<br/></span>
                        <ul class="ul-text">
                            <li>Developed a Django web-interface search engine for hundreds of internal sequencing results produced by all Ion Torrent R&amp;D across the country, increasing ease of access and accepted as an integral tool for comparative studies throughout the company</li>
                            <li>Developed a variety of C, Python, and Perl programs for customer facing Torrent Suite plugins. Utilized JIRA software tracking and version control for all products</li>
                            <li>Supported scientists with custom pipelines to analyze sequence quality and mapping statistics for deficiencies in sequencing protocols and the Ion Torrent AmpliSeq primer design algorithm, including GC bias, primer-strand preferences, targeted sequencing efficacy</li>
                        </ul><br/>

                        <b>Bioinformatics and Molecular Biology Internship</b><span class="dates">05/11-08/11</span><br/>
                        <span class="jobpos">Life Technologies, Ion Torrent R&amp;D and SOLiD Collaborations<br/></span>
                        <ul class="ul-text">
                            <li>Created and presented analyses measuring the quality of sequencing experiments testing various sequencing polymerase mutants for longer reads and higher Phred quality scores</li>
                            <li>Led and supervised sequencing and mapping runs that produced variant calls, mapping statistics, and mapping quality summaries for the SOLiD Collaborations team</li>
                            <li>Assisted team members with sequencing runs, prepared and initialized PGM sequencing instruments, and resolved server-related issues to improve the flow of experiments</li>
                        </ul><br/>

                        <b>Undergraduate Research Assistant</b><span class="dates">05/09-05/11</span><br/>
                        <span class="jobpos">Robert Edwards Bioinformatics Lab, San Diego State University<br/></span>
                        <ul class="ul-text">
                            <li>Developed several Java programs interfacing with a web service API for the SEED genome annotation database</li>
                            <li>Constructed a search hash-algorithm in Java to accurately identify DNA sequence similarity between multiple sequence files</li>
                            <li>Co-developed a front-end Android app producing figures and statistics for the Real Time Metagenomics server-side annotation tool</li>
                            <li>Developed a social network web application to access and distribute metagenomics data while leveraging the network’s API and infrastructure to connect with colleagues and collaborators</li>
                        </ul>
                    </div>

                    <h3>FIRST AUTHOR PUBLICATIONS</h3>
                    <div class="text1">
                        <ol>
                            <li><b>Cuevas, D.A.</b> (2018). <i>Bridging the genomic gaps: genome-scale metabolic network tools for bioinformatics analyses. Dissertation.</i> ProQuest Dissertations Publishing. 10749152.<a href="https://www.proquest.com/openview/1998f8125780a6dec6391e41194821ec" target="_blank">Link</a></li><br/>
                            <li><b>Cuevas, D.A.</b> and Edwards, R.A. (2018). Growth Score:<i> a single metric to define growth in 96-well phenotype assays. PeerJ </i> 6:e4681. <a href="https://doi.org/10.7717/peerj.4681" target="_blank">DOI: 10.7717/peerj.4681.</a></li><br/>
                            <li><b>Cuevas, D.A.</b> and Edwards, R.A. (2017). <i>PMAnalyzer: a new web interface for bacterial growth curve analysis. Bioinformatics</i> 33(12), 1905-1906. <a href="https://doi.org/10.1093/bioinformatics/btx084" target="_blank">DOI: 10.1093/bioinformatics/btx084.</a></li><br/>
                            <li><b>Cuevas, D.A.,</b> Edirisinghe, J., Henry, C.S., Overbeek, R., O’Connell, T.G., and Edwards, R.A. (2016). <i> From DNA to FBA: How to Build Your Own Genome-Scale Metabolic Model. Systems Microbiology,</i> 907. <a href="https://doi.org/10.3389/fmicb.2016.00907" target="_blank">DOI: 10.3389/fmicb.2016.00907.</a></li><br/>
                            <li><b>Cuevas, D.A.,</b> Garza, D., Sanchez, S.E., Rostron, J., Henry, C.S., Vonstein, V., Overbeek, R.A., Segall, A., Rohwer, F., Dinsdale, E.A., and Edwards, R.A. (2014). <i>Elucidating genomic gaps using phenotypic profiles. F1000Research</i> 3, 210. <a href="https://doi.org/10.12688/f1000research.5140.2" target="_blank">DOI: 10.12688/f1000research.5140.2.</a></li>
                        </ol>
                    </div>

                    <h3>CO-AUTHOR PUBLICATIONS</h3>
                    <div class="text1">
                        <ol>
                            <li>Boling, L., <b>Cuevas, D.A.,</b> Grasis, J.A., Kang, H.S., Knowles, B., Levi, K., Maughan, H., McNair, K., Rojas, M.I., Sanchez, S.E., Smurthwaite, C., and Rohwer, F. (2020). <i>Dietary Prophage Inducers and Antimicrobials: Towards Landscaping the Human Gut Microbiome. Gut Microbes,</i> 11:4, 721-734. <a href="https://doi.org/10.1080/19490976.2019.1701353" target="_blank">DOI: 10.1080/19490976.2019.1701353.</a></li><br/>
                            <li>Kuang, E., Marney, M., <b>Cuevas, D.A.,</b> Edwards, R.A., and Forsberg, E.M. (2020). <i>Towards Predicting Gut Microbial Metabolism: Integration of Flux Balance Analysis and Untargeted Metabolomics. Metabolomics,</i> 10(4):156. <a href="https://doi.org/10.3390/metabo10040156" target="_blank">DOI: 10.3390/metabo10040156.</a></li><br/>
                            <li>Edwards R.A., ..., and Dutilh, B.E. (2019). <i>Global phylogeography and ancient evolution of the widespread human gut virus crAssphage. Nature Microbiology,</i> 4, 1727–1736. <a href="https://doi.org/10.1038/s41564-019-0494-6" target="_blank">DOI: 10.1038/s41564-019-0494-6.</a></li><br/>
                            <li>Cobián Güemes, A.G., Lim Y.W., Quinn, R.A., Conrad, D.J., Benler, S., Maughan, H., Edwards, R.A., Brettin, T., Cantú, V.A., <b>Cuevas, D.A.,</b> Hamidi, R., Dorrestein, P., and Rohwer, F. (2019). <i>Cystic Fibrosis Rapid Response: Translating Multi-omics Data into Clinically Relevant Information, mBio,</i> 10:e00431-19. <a href="https://doi.org/10.1128/mBio.00431-19" target="_blank">DOI: 10.1128/mBio.00431-19.</a></li><br/>
                            <li>Silveira, C.B., Coutinho, F.H., Cobián-Güemes, A.G., Cantu, V.A., <b>Cuevas, D.A.,</b> Knowles, B., Cavalcanti, G.S., Soares, A.C., Silva, G.G.Z., Doane, M., Francini-Filho, R., Thompson, C., Dinsdale, E., Edwards, R., Gregoracci, G., Rohwer, F., and Thompson, F. (2018). <i>Phage-mediated transfer of bacterial virulence genes in coral reefs.</i> (In submission).</li><br/>
                            <li>Silva, G.G.Z., Haggerty, J.M., <b>Cuevas, D.A.,</b> Doane, M., Dinsdale, E.A., Dutilh, B.E., and Edwards, R.A. (2018). <i>Ecological Implications of Metagenomics Data Analysis.</i> (In submission).</li><br/>
                            <li>Sczyrba, A., Hofmann, P., Belmann, P., Koslicki, D., Janssen, S., Dröge, J., Gregor, I., Majda, S., Fiedler, J., Dahms, E., Bremges, A., Fritz, A., Garrido-Oter, R., Jørgensen, T.S., Shapiro, N., Blood, P.D., Gurevich, A., Bai, Y., Turaev, D., DeMaere, M.Z., Chikhi, R., Nagarajan, N., Quince, C., Meyer, F., Balvočiūtė, M., Hansen, L.H., Sørensen, S.J., Chia, B.K.H., Denis, B., Froula, J.L., Wang, Z., Egan, R., Kang, D.D., Cook, J.J., Deltel, C., Beckstette, M., Lemaitre, C., Peterlongo, P., Rizk, G., Lavenier, D., Wu, Y.W., Singer, S.W., Jain, C., Strous, M., Klingenberg, H., Meinicke, P., Barton, M.D., Lingner, T., Lin, H.H., Liao, Y.C., Silva, G.G.Z., <b>Cuevas, D.A.,</b> Edwards, R.A., Saha, S., Piro, V.C., Renard, B.Y., Pop, M., Klenk, H.P., Göker, M., Kyrpides, N.C., Woyke, T., Vorholt, J.A., Schulze-Lefert, P., Rubin, E.M., Darling, A.E., Rattei, T., and McHardy, A.C. (2017). <i>Critical Assessment of Metagenome Interpretation − a benchmark of computational metagenomics software. Nature Methods</i> 14(11), 1063-1071. <a href="https://doi.org/10.1038/nmeth.4458" target="_blank">DOI: 10.1038/nmeth.4458.</a></li><br/>
                            <li>Kang, H.S., McNair, K., <b>Cuevas, D.,</b> Bailey, B., Segall, A.M., and Edwards, R.A. (2017). <i>Prophage genomics reveals patterns in phage genome organization and replication. bioRxiv,</i> 114819. <a href="https://doi.org/10.1101/114819" target="_blank">DOI: 10.1101/114819.</a></li><br/>
                            <li>Sanchez, S.E., <b>Cuevas, D.A.,</b> Rostron, J.E., Liang, T.Y., Pivaroff, C.G., Haynes, M.R., Nulton, J., Felts, B., Bailey, B.A., Salamon, P., Edwards, R.A., Burgin, A.B., Segall, A.M., and Rohwer, F. (2015). <i>Phage Phenomics: Physiological Approaches to Characterize Novel Viral Proteins. Journal of Visualized Experiments,</i> e52854. <a href="https://doi.org/10.3791/52854" target="_blank">DOI: 10.3791/52854.</a></li><br/>
                            <li>Lim, Y.W., <b>Cuevas, D.A.,</b> Silva, G.G.Z., Aguinaldo, K., Dinsdale, E.A., Haas, A.F., Hatay, M., Sanchez, S.E., Wegley-Kelly, L., Dutilh, B.E., Harkins, T.T., Lee, C.C., Tom, W., Sandin, S.A., Smith, J.E., Zgliczynski, B., Vermeij, M.J.A., Rohwer, F., and Edwards, R.A. (2014). <i>Sequencing at sea: challenges and experiences in Ion Torrent PGM sequencing during the 2013 Southern Line Islands Research Expedition. PeerJ</i> 2, e520. <a href="https://doi.org/10.7717/peerj.520" target="_blank">DOI: 10.7717/peerj.520.</a></li><br/>
                            <li>Dutilh, B.E., Thompson, C.C., Vicente, A.C., Marin, M.A., Lee, C., Silva, G.G., Schmieder, R., Andrade, B.G., Chimetto, L., <b>Cuevas, D.,</b> Garza, D.R., Okeke, I.N., Aboderin, A.O., Spangler, J., Ross, T., Dinsdale, E.A., Thompson, F.L., Harkins, T.T., and Edwards, R.A. (2014). <i>Comparative genomics of 274 Vibrio cholerae genomes reveals mobile functions structuring three niche dimensions. BMC Genomics</i> 15, 654. <a href="https://doi.org/10.1186/1471-2164-15-654" target="_blank">DOI:10.1186/1471-2164-15-654.</a></li><br/>
                            <li>Silva, G.G.Z., <b>Cuevas, D.A.,</b> Dutilh, B.E., and Edwards, R.A. (2014). <i>FOCUS: an alignment-free model to identify organisms in metagenomes using non-negative least squares. PeerJ</i> 2, e425. <a href="https://doi.org/10.7717/peerj.425" target="_blank">DOI: 10.7717/peerj.425.</a></li><br/>
                            <li>Disz, T., Akhter, S., <b>Cuevas, D.,</b> Olson, R., Overbeek, R., Vonstein, V., Stevens, R., and Edwards, R.A. (2010). <i>Accessing the SEED Genome Databases via Web Services API: Tools for Programmers. BMC Bioinformatics</i> 11, 319. <a href="https://doi.org/10.1186/1471-2105-11-319" target="_blank">DOI:10.1186/1471-2105-11-319.</a></li>
                        </ol>
                    </div>

                    <h3>AWARDS</h3>
                    <div class="text1">
                        <b>14th Annual SDSU Applied Computational Science and Engineering Student Support Poster Award</b><span class="dates">04/17</span><br/>
                        Tioga Research, Inc.<br/><br/>

                        <b>ARCS Foundation Scholar, Two Years</b><span class="dates">09/15-05/17</span><br/>
                        ARCS San Diego Chapter<br/><br/>

                        <b>2014 GEN TEN Award</b><span class="dates">06/14</span><br/>
                        Genetic Engineering and Biotechnology News Publisher<br/><br/>

                        <b>Graduate S-STEM Scholarship, Two Years</b><span class="dates">01/14-01/16</span><br/>
                        National Science Foundation via SDSU Computational Science Research Center<br/><br/>

                        <b>11th Annual Rocky Mountain Bioinformatics Conference Poster Prize</b><span class="dates">12/13</span><br/>
                        F1000Research<br/><br/>

                        <b>Statistical Biomedical Informatics Scholarship, Two Semesters</b><span class="dates">08/12-05/13</span><br/>
                        Biomedical Informatics Research Center, San Diego State University<br/><br/>

                        <b>NIH New Interdisciplinary Workforce Fellow</b><span class="dates">07/09-07/10</span><br/>
                        Computational Science Research Center, San Diego State University<br/><br/>

                        <b>Dean's Honors List, Six Semesters</b><span class="dates">08/08-05/11</span><br/>
                        San Diego State University<br/>
                    </div>

                    <h3>MEMBERSHIPS</h3>
                    <div class="text1">
                        Sigma Xi Scientific Society - Full member<span class="dates">Since 05/16</span><br/>
                        Society for Industrial and Applied Mathematics<span class="dates">Since 10/15</span><br/>
                        San Diego State University Science Alumni Network<span class="dates">Since 05/15</span><br/>
                        San Diego State University Biomedical Technology Student Association - Webmaster<span class="dates">Since 09/12</span><br/>
                    </div>

                    <h3>PEER REVIEWER</h3>
                    <div class="text1">
                        PLoS One<br/>
                    </div>

                    <h3>TALKS AND POSTER PRESENTATIONS</h3>
                    <div class="text1">
                        <b>Unraveling metabolism network models of kelp-associated bacteria</b><span class="dates">03/18</span><br/>
                        (Talk) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>Metabolic model and data-driven exploration linking genotypes to phenotypes</b><span class="dates">07/17</span><br/>
                        (T) SDSU Computational Science Research Center Colloquium, San Diego, CA<br/><br/>
                        <b>Metabolic model and data-driven exploration linking genotypes to phenotypes</b><span class="dates">04/17</span><br/>
                        (Poster) SDSU Applied Computational Science and Engineering Student Support, San Diego, CA<br/><br/>
                        <b>Metabolic model and data-driven exploration linking genotypes to phenotypes</b><span class="dates">04/17</span><br/>
                        (P) ARCS Scientist of the Year Award Dinner, San Diego, CA<br/><br/>
                        <b>Metabolic model and data-driven exploration linking genotypes to phenotypes</b><span class="dates">03/17</span><br/>
                        (P) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>From DNA to FBA: how to build your own genome-scale metabolic model</b><span class="dates">04/16</span><br/>
                        (P) SDSU Applied Computational Science and Engineering Student Support, San Diego, CA<br/><br/>
                        <b>From DNA to FBA: how to build your own genome-scale metabolic model</b><span class="dates">04/16</span><br/>
                        (P) ARCS Scientist of the Year Award Dinner, San Diego, CA<br/><br/>
                        <b>From DNA to FBA: how to build your own genome-scale metabolic model</b><span class="dates">03/16</span><br/>
                        (T) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>Phenotyping Diverse Bacteria for Metabolic Network Reconstruction</b> <span class="dates">06/15</span><br/>
                        (P) General Meeting of the American Society for Microbiology, New Orleans, LA<br/><br/>
                        <b>Phenotyping Diverse Bacteria for Metabolic Network Reconstruction</b> <span class="dates">04/15</span><br/>
                        (P) SDSU Applied Computational Science and Engineering Student Support, San Diego, CA<br/><br/>
                        <b>Phenotyping Diverse Bacteria for Metabolic Network Reconstruction</b> <span class="dates">03/15</span><br/>
                        (T) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>High-Throughput Phenotype Profiling for Bacterial Flux-Balance Model Optimization</b> <span class="dates">06/14</span><br/>
                        (T) Biotechnology Industry Organization International Convention, San Diego, CA<br/><br/>
                        <b>High-Throughput Phenotype Profiling for Bacterial Flux-Balance Model Optimization</b> <span class="dates">05/14</span><br/>
                        (P) General Meeting of the American Society for Microbiology, Boston, MA<br/><br/>
                        <b>Metabolic model optimization using high-throughput phenotypic assays</b> <span class="dates">04/14</span><br/>
                        (T) Diponegoro University, Semarang, Indonesia<br/><br/>
                        <b>High-Throughput Phenotype Profiling for Bacterial Flux-Balance Model Optimization</b> <span class="dates">03/14</span><br/>
                        (T) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>High-Throughput Phenotype Profiling for Bacterial Flux-Balance Model Optimization</b> <span class="dates">01/14</span><br/>
                        (P) Southern California Systems Biology Conference, Irvine, CA<br/><br/>
                        <b>High-Throughput Phenotype Profiling for Bacterial Flux-Balance Model Optimization</b> <span class="dates">12/13</span><br/>
                        (T &amp; P) Rocky Mountain Bioinformatics Conference, Snowmass, CO<br/><br/>
                        <b>Measuring Growth Quality of Bacteria Using Phenotypic Microarrays</b> <span class="dates">05/13</span><br/>
                        (P) San Diego Microbiology Group All-day Meeting, San Diego, CA<br/><br/>
                        <b>Characterizing Phenotypic Properties of Unexplored Viral Genes</b> <span class="dates">03/13</span><br/>
                        (T) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>Compiling Genome Metadata for Comparative Analysis</b> <span class="dates">12/12</span><br/>
                        (T) Rocky Mountain Bioinformatics Conference, Snowmass, CO<br/><br/>
                        <b>Rapid Sequence Searching Using Hashing</b> <span class="dates">03/11</span><br/>
                        (T) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                        <b>Real-Time Metagenomics</b> <span class="dates">05/10</span><br/>
                        (P) General Meeting of the American Society for Microbiology, San Diego, CA<br/><br/>
                        <b>Real-Time Metagenomics</b> <span class="dates">05/10</span><br/>
                        (P) San Diego Microbiology Group All-day Meeting, San Diego, CA<br/><br/>
                        <b>Real-Time Metagenomics</b> <span class="dates">03/10</span><br/>
                        (P) SDSU Applied Computational Science and Engineering Student Support, San Diego, CA<br/><br/>
                        <b>Real-Time Metagenomics</b> <span class="dates">03/10</span><br/>
                        (T) San Diego State University Student Research Symposium, San Diego, CA<br/><br/>
                    </div>

                    <h3>LECTURES</h3>
                    <div class="text1">
                        Guest lecture at San Diego State University. Course: BIOL549 - Microbial Genetics and Physiology<span class="dates">11/16</span><br/>
                        <i>Flux-balance analysis using genome-scale metabolic models</i><br/><br/>

                        Guest speaker at the ARCS General Meeting, Rancho Sante Fe, CA<span class="dates">06/16</span><br/>
                        <i>Metabolic modeling in the era of genomics</i><br/><br/>

                        Summer workshop at San Diego State University<span class="dates">05/16 - 08/16</span><br/>
                        <i>Introduction to Python programming for microbiology scientists</i><br/><br/>
                    </div>
                </div>
            </div>
        </div>
        <hr>
        <div class="container footer">
            <div class="row">
                <div class="col-xs-6" style="text-align:left">
                    <p>Copyright &copy; Daniel Cuevas
                    <script type="text/javascript">
                        document.write(new Date().getFullYear());
                    </script>
                    </p>
                </div>
                <div class="col-xs-6" style="text-align:right">
                    <ul class="list-inline">
                        <li>
                            <a href="https://twitter.com/Daniel_A_Cuevas"><i class="fa fa-twitter fa-2x"></i></a>
                        </li>
                        <li>
                            <a href="http://www.linkedin.com/in/cuevasdaniel"><i class="fa fa-linkedin fa-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://scholar.google.com/citations?user=E_Qlo2UAAAAJ&hl=en"><i class="ai ai-google-scholar ai-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://github.com/dacuevas/"><i class="fa fa-github fa-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://www.researchgate.net/profile/Daniel_Cuevas2"><i class="ai ai-researchgate ai-2x"></i></a>
                        </li>
                    </ul>
                </div>
            </div>
            <!-- /.row -->
        </div>
    </header>

    <!-- Content Section -->
                    <!--
    <section>
        <div class="container">
            <div class="row">
                <div class="col-xs-12">
                    <h1 class="section-heading">Section Heading</h1>
                    <p class="lead section-lead">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
                    <p class="section-paragraph">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid, suscipit, rerum quos facilis repellat architecto commodi officia atque nemo facere eum non illo voluptatem quae delectus odit vel itaque amet.</p>
                    <h1 class="section-lead" id="About">About Me</h1>
                    <p class="section-paragraph" style="font-style:italic"># site under construction</p>
                    <p class="section-paragraph">In the meantime, check me out at my social media links below!</p>
                </div>
            </div>
        </div>
    </section>

                    -->

    <!-- Footer -->
    <!--
    <footer>
        <div class="container">
            <div class="row">
                <div class="col-xs-6">
                    <p>Copyright &copy; Daniel Cuevas 2016</p>
                </div>
                <div class="col-xs-6" style="text-align:right">
                    <ul class="list-inline">
                        <li>
                            <a href="http://www.linkedin.com/in/cuevasdaniel"><i class="fa fa-linkedin fa-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://scholar.google.com/citations?user=E_Qlo2UAAAAJ&hl=en"><i class="ai ai-google-scholar ai-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://www.researchgate.net/profile/Daniel_Cuevas2"><i class="ai ai-researchgate ai-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://github.com/dacuevas/"><i class="fa fa-github fa-2x"></i></a>
                        </li>
                        <li>
                            <a href="https://twitter.com/Daniel_A_Cuevas"><i class="fa fa-twitter fa-2x"></i></a>
                        </li>
                    </ul>
                </div>
            </div>
            <!-- /.row -->
        </div>
        <!-- /.container -->
    </footer>
    <!--
    -->
    <!-- jQuery -->
    <script src="assets/js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="assets/js/bootstrap.min.js"></script>

</body>

</html>
